spec/cluster_spec.rb

```diff
@@ -5,8 +5,8 @@
 
```

Comment:
This was done to fix a once-in-1000 brittle test, not directly related to the ticket.